lib/croutons/breadcrumb_trail.rb

```diff
@@ -15,16 +15,16 @@ def initialize(template_identifer, objects = {})
       build_breadcrumbs
     end
 
-    def method_missing(name, *args)
+    def method_missing(name, *args, &block)
```

Comment:
I'd argue that an unused block argument reduces clarity, because any Ruby method can take a block (even if it doesn't explicitly have an `&block` argument), so the only reason to declare the argument is the block is passed on to some other method.